docs/process/prd.md

```diff
@@ -0,0 +1,226 @@
+
+ICU4X 0.1’s selection of components aimed to:
+
+-   Validate low-level models around [data management](https://github.com/unicode-org/icu4x/blob/master/docs/design/data_pipeline.md)
+-   Establish project culture using simple building blocks such as [Locale](https://docs.rs/icu_locid/0.1.0/icu_locid/) and [Plural Rules](https://docs.rs/icu_plurals/0.1.0/icu_plurals/)
+-   Introduce a low level foundation for string operations with [Unicode Set](https://docs.rs/icu_datetime/0.1.0/icu_datetime/)
+-   Expose a single, high-level, highly requested API - [DateTimeFormat](https://docs.rs/icu_datetime/0.1.0/icu_datetime/)
+-   Release a meta-package [ICU](https://docs.rs/icu/0.1.0/icu/)
+
+With the upcoming 0.2 release, we aim to close the gap between our 0.1 features and requirements of ECMA-402.
+
+## Risk Analysis
+
+Developing a high quality solution for the internationalization industry needs is a noble goal which requires a lot of time to design correctly. All three current stakeholders are well positioned to justify such effort.
+
+This allows ICU4X to strive for project quality rather than short-term business needs, but software projects disconnected from business needs are at risk of developing the [“Ivory Tower”](https://medium.com/it-dead-inside/knocking-down-the-ivory-tower-72fd249a8db7) syndrome which exposes risk of disconnection from real world alignment.
+
+We strive therefore to balance the roadmap between focusing on industry level excellence with business alignments which allow us to frequently evaluate our ability to deliver on the value proposition and recognize early if the effort is not yielding expected results.
+
+To achieve that, we identified a number of milestones, aligned with project planning, at which we’ll be able to test the prototypes of ICU4X against real business needs, validate and learn from the tests and adapt our roadmap and planning to ensure ICU4X fits business requirements and has an answer to actual product needs.
+
+## Business Alignments
+
+We identified a number of opportunities to evaluate progress of ICU4X on the path to 1.0 release against production environments:
+
+### Test262
+
+With the upcoming 0.2 release, ICU4X will provide at least three components that can be validated against a comprehensive internationalization test harness - [test262](https://github.com/tc39/test262) - used by ECMA-402:
+
+-   [Intl.Locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale)
+-   [Intl.PluralRules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules)
+-   [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat)
+
+The former two should be able to pass the full test scope.
```

Comment:
The 262 test suite has lots of JS specific elements, like is returned value an Object/Array etc. I assume we are not interested in fulfilling those.

Reply:
We are not, but I assume we want sticking ICU4X into a JS implementation (SpiderMonkey, Deno, V8 etc.) ergonomic so that they only have to care about Object/Array etc.

Do you think it should be stated explicitly here? If so, do you have a suggestion how to phrase it?

Reply:
I think you should commit it as is, and I'll edit (with your review) afterwards?